pages/common/cs-install.md

```diff
@@ -0,0 +1,36 @@
+# cs install
+
+> Install an application in the installation directory onfigured when installing `cs`  (to enable the binary to be loaded add to your `.bash_profile` the `$ eval "$(cs install --env)"` command).
+> More information: <https://get-coursier.io/docs/cli-install>.
+
+- Install an application:
+
+`cs install {{application-name}}`
+
+- Install a specific version of an application:
+
+`cs install {{application-name}}:{{application-version}}`
+
+- Search an application by name:
+
+`cs search {{application-partial-name}}`
+
+- Update an application if available:
+
+`cs update {{application-name}}`
```

Comment:
```suggestion
- Update a specific application if available:

`cs update {{application_name}}`
```

Almost the same issue as above.
Mandatory fix: [unexpected `application-name` placeholder content, `_` expected](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).